pkg/kubelet/cm/cpumanager/policy_static_test.go

```diff
@@ -1059,6 +1062,99 @@ func TestStaticPolicyStartWithResvList(t *testing.T) {
 	}
 }
 
+func TestStaticPolicyNoSMT(t *testing.T) {
```

Comment:
this should be more like `TestMetricsAllocate` I think.
Furthermore, I'd move this new test in its own new file `metrics_test.go`

Reply:
Agreed. Will adjust as you suggested.